install-dependencies.sh

```diff
@@ -142,6 +142,7 @@ declare -A pip_packages=(
     [treelib]=
     [allure-pytest]=
     [pytest-xdist]=
+    [universalasync]=
```

Comment:
IIUC, changing install-dependencies.sh should be done in a separate patch because @avikivity needs to rebuild the dbuild system once its done.
In that patch, please also explain why you need this "universalasync" package (which I never heard of).

Reply:
If we apply such cleanups during the migration, it will never end.

Reply:
yes, it needs to be in a separate patch in a separate pull requet